tests/testthat/test-hazardFunctions.R

```diff
@@ -14,6 +14,18 @@ test_that("ExpHazOS works also with vector of times t", {
   expect_equal(result, expected, tolerance = 1e-3)
 })
 
+# avgHRInteg ----
```

Comment:
sorry, last comment - would it be better to call this `avgHRIntegExpOS`? since we might have others later for the other distributions?

Reply:
(Note that in RStudio you can replace this across files with `CTRL + Shift + F` and then replace)

Reply:
Thx, That`s very cool, didn`t know that!!